### DISCUSSION TITLE
PostgreSQL version compatability

### DISCUSSION THREAD

**Discussion by kript** [UtrechtUniversity/yoda#198] [Q&A] [answered]
Hi folks,

we have a Yoda instance for testing, which points to an external PostgreSQL database. Is there a compatability matrix for what versions of PostgreSQL are OK? This for other than iRODS related functions, as I have asked RENCI the same question. However, it would be good to know per iRODS release what versions are supported.

thanks,

John

**Comment by stsnel** [comment, marked as answer]
Hi John,

As of Yoda 1.8, we only test and deploy Yoda with PostgreSQL 9.2.x (the version bundled with CentOS 7) for both the iCAT and EUS database. I'm not aware of any limitations regarding using newer PostgreSQL versions in an external database, as long as it's compatible with iRODS.

We're in the process of testing a new version of the playbook that installs PostgreSQL from the PostgreSQL repositories, so that admins have more flexibility regarding which database version to use for the iCAT database. This option to specify the database version to install is expected to be available from Yoda 1.9 onwards. We aim to deploy and test with PostgreSQL 11 (or possibly a later version) by default in Yoda 1.9.

Best,
Sietse

**Comment by fiberbit** [comment]
As of Yoda 1.8.7 Yoda can use PostgreSQL 15, and is also using this in production without problems.
https://utrechtuniversity.github.io/yoda/administration/upgrading-postgresql.html
